README.md

```diff
@@ -0,0 +1,3 @@
+# Tangobookings
+
+backpage for tango taxi dancers and teachers
```

Comment:
Should be capitalized and also contain instructions for how to deploy the app **locally**.